source/llvm.natvis

```diff
@@ -187,14 +187,6 @@ For later versions of Visual Studio, no setup is required.
     <DisplayString>{Data}</DisplayString>
   </Type>
 
-  <Type Name="llvm::Optional&lt;*&gt;">
-    <DisplayString Condition="!Storage.hasVal">None</DisplayString>
-    <DisplayString Condition="Storage.hasVal">{Storage.value}</DisplayString>
```

Comment:
LOL don't change the LLVM natvis file, it is copied from the llvm repo. In fact we should remove the natvis file from our repo, and change the CMakeLists.txt to reference the copy of the natvis file that is in the llvm installation.

Reply:
oh LOL my bad